integreat_cms/cms/templates/dashboard/dashboard.html

```diff
@@ -19,7 +19,9 @@ <h1 class="heading">{% translate "My Dashboard" %}</h1>
                 <!-- App Size Widget: TODO -->
                 <!-- Page HIX Widget-->
                 {% if TEXTLAB_API_ENABLED and request.region.hix_enabled %}
```

Comment:
```suggestion
                {% if TEXTLAB_API_ENABLED and request.region.hix_enabled and worst_hix_translations %}
```
We could include the condition in this line